docs/docs/reference/metaprogramming/tasty-reflect.md

```diff
@@ -44,7 +44,7 @@ def natConstImpl(x: Expr[Int])(using Quotes): Expr[Int] = {
   import quotes.reflect._
   val xTree: Term = x.asTerm
   xTree match {
-    case Inlined(_, _, Literal(Constant(n: Int))) =>
+    case Inlined(_, _, Literal(Constant.Int(n))) =>
```

Comment:
```suggestion
    case Inlined(_, _, Literal(IntConstant(n))) =>
```

Reply:
Thanks! I tested it with M2, but wasn't sure about the current state of affairs.

Reply:
That one changed in M3 3 days ago. Easy to miss.